copier.yml

```diff
@@ -1,25 +1,37 @@
-_imports:
-  parse: packaging.version.parse
```

Comment:
Wasn't this working?

Reply:
No, I couldn't find `_imports` documented anywhere in `copier`'s documentation. And it turns out that YAML fields beginning with `_` will be ignored during the actual copy interaction when it asks for values for the fields.